README.md

```diff
@@ -39,12 +39,18 @@ This tool does cleaning the support zip by
 - extracting the zip file
 - replacing in files (defined via LOGDIR, default: all):
 
-|Search-String|Replacement|
+|Example|Replacement|
 |---|---|
-|_BASEURL_|URL_CLEANED|
+|subdomain._BASEURL_|URL_CLEANED|
 |userName: _NAME_WITHOUT_SPACE_|userName: USERNAME_CLEANED|
-
-(_NAME_WITHOUT_SPACE_ matches on all not white space characters)
+| customer@business.com | EXTERNAL_EMAIL_SHA256:b6c4586387_CLEANED|
+| employee@URL_CLEANED | INTERNAL_EMAIL_SHA256:2fdc017705_CLEANED|
+| *.smhss.de | SMEDIA_DOMAIN_CLEANED|
+| 127.0.0.1 | 127.0.0.IP_CLEANED|
+| ----- BEGIN RSA PRIVATE KEY ----- ... | PRIVATE_KEY_CLEANED|
+| ----- BEGIN CERTIFICATE ----- ... | CERTIFICATE_CLEANED|
+| Big Business LLC | BUSINESS_CLEANED|
+| Herr Müller | NAME_CLEANED |
 
 - remove files that are older than DELETE_AFTER_DAYS (default: 180) days (if --delete-oldest is set)
```

Comment:
Die delete-oldest and delete-biggest flags gibts nicht mehr, das sollten wir noch anpassen. Ich glaub aber auch, dass wir die nicht brauchen

Reply:
Ganz andere Sache, aber gibt gerade keinen Ort, wo man das hinschreiben kann.
_list_files_in_dir gehört wohl eher zu den helper functions als zum zip createn

ggf kann man da noch mal grob die Reihenfolge durchgucken